boot.php

```diff
@@ -15,7 +15,12 @@
     throw new RuntimeException('You need to pass the ROM file name (Ex: drmario.rom)');
 }
 
-$rom = file_get_contents('roms/' . $argv[1]);
+$filename = 'roms/'.$argv[1];
```

Comment:
I think you should not do that.

You make it available on Packagist, and people can install it by:

```bash
composer global require gabrielrcouto/php-gameboy
```

Doing that, people can run any ROM in any place.